packages/clerk-js/webpack.config.js

```diff
@@ -29,7 +29,7 @@ const common = ({ mode }) => {
   return {
     mode,
     resolve: {
-      extensions: ['.ts', '.tsx', '.js', '.jsx'],
+      extensions: ['.ts', '.tsx', '.mjs', '.js', '.jsx'],
```

Comment:
Order matters here, so .mjs will be tried before .js

Reply:
Thanks - can we add a comment here as this is easy to miss?

Reply:
Sure!